packages/composables/src/useCart.ts

```diff
@@ -131,6 +137,22 @@ export function useCartFunction(): UseCartReturn {
     return addToCartResult;
   }
 
+  async function addProducts(
```

Comment:
Could we maybe refactor `addProduct` to use this new function inside so we have consistency and less code?

Reply:
This would be a breaking change. I changed `addProduct` to be deprecated.

Reply:
that would not be a breaking change, as the API for `addProduct` would not change, only internally it would call `addProducts` inside